tests/unit/django_spanner/test_operations.py

```diff
@@ -111,67 +113,162 @@ def test_convert_uuidfield_value_none(self):
         )
 
     def test_date_extract_sql(self):
-        self.assertEqual(
-            self.db_operations.date_extract_sql("week", "dummy_field"),
-            "EXTRACT(isoweek FROM dummy_field)",
-        )
+        if USING_DJANGO_3:
+            self.assertEqual(
+                self.db_operations.date_extract_sql("week", "dummy_field"),
+                "EXTRACT(isoweek FROM dummy_field)",
+            )
+        elif USING_DJANGO_4:
```

Comment:
Here also change to `else` blocks to (hopefully) simplify the next major version bump.

Reply:
Done